backend/package.json

```diff
@@ -0,0 +1,35 @@
+{
+    "name": "backend",
+    "version": "1.0.0",
+    "description": "",
+    "main": "src/app.ts",
+    "scripts": {
+        "tsc": "tsc",
+        "dev": "nodemon src/app.ts",
```

Comment:
does it even work with ts like that?
Also, I recommend `ts-node-dev`

Reply:
If you want absolute imports with TS and Node, check out `tsconfig-paths`, and then
- for dev:
`"ts-node-dev -r tsconfig-paths/register --no-deps --respawn --poll --interval 1000 src/app.ts`
- for prod:
 `tsc` &&
 `start:prod": "node -r ./tsconfig-paths.js dist/app.js`,